rust/bolos-sys/src/pic.rs

```diff
@@ -68,26 +68,117 @@ impl<T> PIC<T> {
         }
     }
 
-    pub fn into_inner(self) -> T {
+    // pub fn into_inner(self) -> T {
```

Comment:
I commented it out because the functionality is incorrect, but didn't want to lose the code since it _could_ become useful.
But I have replaced it with a proper variant anyways